Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.mm

```diff
@@ -265,8 +276,6 @@
         m_videoLayers.remove(videoLayerIter);
     }
 #endif
-
-    m_nodes.remove(layerID);
```

Comment:
Why did this go away?

Reply:
(See the take above)

Reply:
Yeah, this was redundant.  I could've mentioned that in the change log.